src/app/content/studyGuides/selectors.ts

```diff
@@ -47,7 +47,10 @@ export const totalCountsPerPageOrEmpty = createSelector(
 
 export const studyGuidesOpen = createSelector(
   studyGuidesSummary,
-  (summary) => summary.open
+  studyGuidesEnabled,
+  parentSelectors.book,
+  parentSelectors.page,
+  (summary, flagEnabled, book, page) => summary.open && flagEnabled && !!book && !!page
```

Comment:
@TomWoodward initially I checked for a non-empty `summary.studyGuides` but that causes the modal to rerender when filters are changed